pyproject.toml

```diff
@@ -202,3 +197,9 @@ max-complexity = 41
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.ruff.lint.pylint]
+max-args = 26
+max-branches = 48
+max-returns = 7
+max-statements = 151
```

Comment:
Is there somewhere I can see what each of these mean?
I get max-args and max-returns, not so sure about branches and statements

Reply:
added link to the doc for those